### PR TITLE
docs: add documentation

### PR DESCRIPTION
**Pull request by cristianmtr**
- overview
- [x] guide on Indexing vs Query
- [x] dump
- [x] rolling update
- [x] individual docs

### CODE REVIEW

**Review by bwanglzu — COMMENTED** [review]
nice documentation, can we add a table indicates which indexer map to which searcher?

**Review by Yongxuanzhang — APPROVED** [review]
Great work!

**Comment by cristianmtr** [comment]
> nice documentation, can we add a table indicates which indexer map to which searcher?

There is no strict relation. Only between PostgresIndexer and PostgresSearcher

File: jinahub/indexers/cache/README.md [hunk 1]
```diff
@@ -1,10 +1,108 @@
...
+
+### 📦️ Via Pypi
+
+1. Install the `executor-indexers` package.
+
+	```bash
+	pip install git+https://github.com/jina-ai/executor-indexers/
+	```
+
+1. Use `jinahub-MY-DUMMY-EXECUTOR` in your code
```
[the remark this answers is not in the document]
Comment (cristianmtr) [reply jinahub/indexers/cache/README.md]: Good catch. There were still a few left. Updated

**Pull request merged**